src/scene/Scene.cpp

```diff
@@ -183,3 +194,40 @@ void Scene::requestASRebuild() { cachedAS.reset(); }
 void Scene::requestSBTRebuild() { cachedSBT.reset(); }
 
 CudaStream::Ptr Scene::getStream() const { return stream; }
+
+void Scene::setupGASForEntities()
+{
+	for (auto&& entity : entities) {
+		/////////////////////
+		// Animated entity //
```

Comment:
Nice section separation. Do we use it somewhere else?

Reply:
Thanks. It seems I am the precursor.

Reply:
And now seriously.
That's inconsistency.

As far as I remember we have similar cases with much simpler comment formating.

Reply:
> As far as I remember we have similar cases with much simpler comment formating.

Could you give an example or your proposition? I'll appreciate it.

I wanted to separate sections and the steps (and comments) performed within them precisely.

Reply:
Even newer tests which share very similar structure.
Ex:
```
TEST_F(GraphMultiReturn, vlp16_data_compare)
{
	// Lidar
	const std::vector<rgl_mat3x4f> raysTf{Mat3x4f::TRS({0.0f, 0.0f, 0.0f}, {90.0f, 0.0f, -90.0f}).toRGL()};
	const float lidarCubeFaceDist = vlp16LidarObjectDistance;
	const float lidarCubeCenterDist = lidarCubeFaceDist + CUBE_HALF_EDGE;
	const auto lidarTransl = Vec3f{lidarCubeCenterDist, 0.0f, 0.0f};
	const rgl_mat3x4f lidarPose = Mat3x4f::TRS(lidarTransl).toRGL();


	// Scene
	spawnCubeOnScene(Mat3x4f::identity());


	constructMRGraph(raysTf, lidarPose, vlp16LidarHBeamDivergence, vlp16LidarVBeamDivergence);


	EXPECT_RGL_SUCCESS(rgl_graph_run(mrRays));
```

Reply:
The simple comment doesn't separate sections and steps well. It would result as follows:
```
		// Non-animated entity
		// Skip if GAS already built
		if (gasBuilderForEntities.contains(entity)) {
			continue;
		}

		// Build GAS if static mesh doesn't have it yet
		const auto& mesh = entity->mesh;
		if (!gasBuilderForStaticMeshes.contains(mesh)) {
			gasBuilderForStaticMeshes[mesh] = std::make_shared<GASBuilder>(getStream(), mesh->dVertices, mesh->dIndices);
		}
		// Assign static mesh GAS to the Entity
		gasBuilderForEntities[entity] = gasBuilderForStaticMeshes[mesh];
```
Here, `Non-animated entity` section comment is on the same "level" as the rest of the comments describing steps performed within this section.